src/modules/ekf2/EKF/aid_sources/sideslip/sideslip_fusion.cpp

```diff
@@ -64,16 +64,11 @@ void Ekf::controlBetaFusion(const imuSample &imu_delayed)
 			updateSideslip(_aid_src_sideslip);
 			_innov_check_fail_status.flags.reject_sideslip = _aid_src_sideslip.innovation_rejected;
 
-			// If starting wind state estimation, reset the wind states and covariances before fusing any data
-			if (!_control_status.flags.wind) {
-				// activate the wind states
-				_control_status.flags.wind = true;
-				// reset the timeout timers to prevent repeated resets
-				_aid_src_sideslip.time_last_fuse = imu_delayed.time_us;
-				resetWindToZero();
+			if (!fuseSideslip(_aid_src_sideslip) && !_control_status.flags.wind && !_external_wind_init) {
+				resetWind();
 			}
 
-			fuseSideslip(_aid_src_sideslip);
+			_control_status.flags.wind = true;
```

Comment:
So even when `fuseSideslip` always returns `false` then `_control_status.flags.wind` is still `true`?